### ISSUE TITLE
[Android] build fails

### ISSUE THREAD

**Issue by sergiobenrocha2** [libretro/mgba#56]
```
/home/buildbot/tools/android/android-ndk-r13b//toolchains/arm-linux-androideabi-4.9/prebuilt/linux-x86_64/bin/arm-linux-androideabi-gcc -c -o ../src/arm/decoder.android_armeabi-v7a.o ../src/arm/decoder.c -fpic -ffunction-sections -funwind-tables -fstack-protector -no-canonical-prefixes -march=armv7-a -mfpu=vfpv3-d16 -mfloat-abi=softfp -fomit-frame-pointer -fstrict-aliasing -funswitch-loops -finline-limit=300 -Wa,--noexecstack -Wformat -Werror=format-security  -DANDROID -DINLINE=inline -DHAVE_STDINT_H -DBSPF_UNIX -DHAVE_INTTYPES -DLSB_FIRST -D__LIBRETRO__ -DMINIMAL_CORE=2 -DM_CORE_GBA -DM_CORE_GB -DHAVE_STDINT_H -DHAVE_INTTYPES_H -DINLINE=inline -DCOLOR_16_BIT -DCOLOR_5_6_5 -DRESAMPLE_LIBRARY=2 -DM_PI=3.14159265358979323846 -DPATH_MAX=4096 -DSSIZE_MAX=32767 -DGIT_VERSION=\"0.5.1\" -DNDEBUG -DUSE_VFS_FILE -DHAVE_LOCALTIME_R  -I/home/buildbot/tools/android/android-ndk-r13b//platforms/android-3/arch-arm/usr/include -I/home/buildbot/tools/android/android-ndk-r13b//sources/cxx-stl/gnu-libstdc++/4.9/include -I/home/buildbot/tools/android/android-ndk-r13b//sources/cxx-stl/gnu-libstdc++/4.9/libs/armeabi/include -I/home/buildbot/tools/android/android-ndk-r13b//sources/cxx-stl/gnu-libstdc++/4.9/include/backward -I../src -I../src/arm -O3 -DNDEBUG -Wall -std=c99
In file included from ../src/arm/arm.h:9:0,
                 from ../src/arm/decoder.h:9,
                 from ../src/arm/decoder-thumb.c:6:
../src/util/common.h:9:19: fatal error: ctype.h: No such file or directory
 #include <ctype.h>
                   ^
In file included from ../src/arm/isa-arm.h:9:0,
                 from ../src/arm/isa-arm.c:6:
../src/util/common.h:9:19: fatal error: ctype.h: No such file or directory
 #include <ctype.h>
                   ^
In file included from ../src/arm/arm.h:9:0,
                 from ../src/arm/arm.c:6:
../src/util/common.h:9:19: fatal error: ctype.h: No such file or directory
 #include <ctype.h>
                   ^
compilation terminated.
compilation terminated.
compilation terminated.
In file included from ../src/arm/arm.h:9:0,
                 from ../src/arm/decoder.h:9,
                 from ../src/arm/decoder-arm.c:6:
../src/util/common.h:9:19: fatal error: ctype.h: No such file or directory
 #include <ctype.h>
                   ^
compilation terminated.
Makefile.rules:20: recipe for target '../src/arm/arm.android_armeabi-v7a.o' failed
In file included from ../src/arm/arm.h:9:0,
                 from ../src/arm/decoder.h:9,
                 from ../src/arm/decoder.c:6:
../src/util/common.h:9:19: fatal error: ctype.h: No such file or directory
 #include <ctype.h>
                   ^
make: *** [../src/arm/arm.android_armeabi-v7a.o] Error 1
compilation terminated.
make: *** Waiting for unfinished jobs....
Makefile.rules:20: recipe for target '../src/arm/isa-arm.android_armeabi-v7a.o' failed
make: *** [../src/arm/isa-arm.android_armeabi-v7a.o] Error 1
Makefile.rules:20: recipe for target '../src/arm/decoder-thumb.android_armeabi-v7a.o' failed
make: *** [../src/arm/decoder-thumb.android_armeabi-v7a.o] Error 1
Makefile.rules:20: recipe for target '../src/arm/decoder-arm.android_armeabi-v7a.o' failed
make: *** [../src/arm/decoder-arm.android_armeabi-v7a.o] Error 1
In file included from ../src/arm/isa-thumb.h:9:0,
                 from ../src/arm/isa-thumb.c:6:
../src/util/common.h:9:19: fatal error: ctype.h: No such file or directory
 #include <ctype.h>
                   ^
compilation terminated.
```

**Comment by sergiobenrocha2** [comment]
I can see 

```
/home/buildbot/tools/android/android-ndk-r13b//toolchains/arm-linux-androideabi-4.9/prebuilt/linux-x86_64/bin/arm-linux-androideabi-gcc

-I/home/buildbot/tools/android/android-ndk-r13b//platforms/android-3/arch-arm/usr/include
-I/home/buildbot/tools/android/android-ndk-r13b//sources/cxx-stl/gnu-libstdc++/4.9/include
-I/home/buildbot/tools/android/android-ndk-r13b//sources/cxx-stl/gnu-libstdc++/4.9/libs/armeabi/include
-I/home/buildbot/tools/android/android-ndk-r13b//sources/cxx-stl/gnu-libstdc++/4.9/include/backward
```

ctype.h is in the first include (platforms/android-3/arch-arm/usr/include), why it's still bullshiting us

**Comment by sergiobenrocha2** [comment]
Wait, there's no android-3 in r13b ndk, it start with android-9. I can find it in r10e.

@fr500 do we really need Android 1.5 support?

**Comment by endrift** [comment]
Wow, I would drop anything below Gingerbread if I were you. I know how much twinaphex likes backcompat, but it's...basically 0% of the userbase that's pre-Gingerbread anymore.

**Comment by twinaphex** [comment]
Android 2.3 is our cutoff point, since that basically allows us to keep running the core on the Xperia Play.

**Comment by twinaphex** [comment]
Android below 2.3 (and below API level 9) wouldn't even be compatible with native activity anyway.

**Comment by sergiobenrocha2** [comment]
Fixed in b77aca9
